frontend/packages/insights-plugin/src/components/InsightsPopup/mappers.ts

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import * as _ from 'lodash';
+
+import {
+  global_palette_blue_50,
+  global_palette_blue_300,
+  global_palette_gold_400,
+  global_palette_orange_300,
+  global_palette_red_200,
+} from '@patternfly/react-tokens';
```

Comment:
Please disable `@typescript-eslint/camelcase` rule only for the relevant imports:

```ts
import * as _ from 'lodash';
/* eslint-disable @typescript-eslint/camelcase */
import {
  global_palette_blue_50,
  global_palette_blue_300,
  global_palette_gold_400,
  global_palette_orange_300,
  global_palette_red_200,
} from '@patternfly/react-tokens';
/* eslint-enable @typescript-eslint/camelcase */
```